### PR TITLE
libfuzzer library integration tests

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

**Comment by bmc-msft** [comment]
This PR is dependant on #682

File: src/integration-tests/libfuzzer-dlopen/main.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#include <dlfcn.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+int (*fuzz_func)(const uint8_t *data, size_t size);
+
+int LLVMFuzzerInitialize(int *argc, char ***argv)
+{
+    printf("initilize\n");
```
Comment (ranweiler) [inline src/integration-tests/libfuzzer-dlopen/main.c:14]:
```suggestion
    printf("initialize\n");
```

File: src/integration-tests/integration-test.py [hunk 2]
```diff
@@ -140,6 +166,18 @@ class Integration(BaseModel):
             ContainerType.coverage: 1,
         },
     ),
+    "windows-libfuzzer-linked-library": Integration(
```
Comment (ranweiler) [inline src/integration-tests/integration-test.py:169]: Can we add a `windows-libfuzzer-loadlibrary` integration test, similar to the `-dlopen` one?

Comment (bmc-msft) [reply src/integration-tests/integration-test.py:169]: Yes, though that's in progress.  I will add that as a follow-on PR.

**Pull request merged**